dscache/convert.go

```diff
@@ -100,8 +100,8 @@ func buildDscacheFlatbuffer(userPairList []userProfilePair, entryInfoList []*ent
 		prettyName := builder.CreateString(ce.Name)
 		metaTitle := builder.CreateString(ce.MetaTitle)
 		themeList := builder.CreateString(ce.ThemeList)
-		commitTitle := builder.CreateString(ce.CommitTitle)
-		commitMessage := builder.CreateString(ce.CommitMessage)
+		// commitTitle := builder.CreateString(ce.CommitTitle)
```

Comment:
I don't feel comfortable with this right now as I have a feeling it goes down the wrong path and will change behavior we don't want to change. Haven't thoroughly explored this yet, but all the changes in dscache feel a bit off right now.

Reply:
It's correct to remove `commitTitle` and `commitMessage` from this location. Only log is using those fields, and no other users from dscache care about them. But, `commitTime` should stay, that's used in other callers.